features/step_definitions/autocomplete_steps.rb

```diff
@@ -41,147 +46,171 @@
     when "Characters"
       "ellen"
     when "Relationships"
-      "stiel"
-    when "Your Tags"
+      "cast"
```

Comment:
I changed "stiel" to "cast". We set up Redis to look for prefixes for each character in a ship ("c", "ca", "cas", etc), searching for a suffix doesn't work.